### PR TITLE
Moving helm chart directory for mysql-exporter

### PR DESCRIPTION
**Pull request by thangarajbluescape**
Please review and merge this PR.

### CODE REVIEW

**Review by ashwinrajadurai3 — APPROVED** [review]
I get it. LGTM

**Comment by notjames** [comment]
I can merge after all the tests succeed. There's one that's failing.

File: helm-chart-sources/mysql-exporter/Chart.yaml [hunk 1]
```diff
@@ -7,5 +7,7 @@ appVersion: v0.12.1
 sources:
   - https://github.com/prometheus/mysqld_exporter
 maintainers:
-  - name: thangaraj ravi
-    email: thangaraj.ravi@bluescape.com
+  - name: juanchimienti
+    email: juan.chimienti@gmail.com
+  - name: monotek
+    email: monotek23@gmail.com
```
Comment (notjames) [inline helm-chart-sources/mysql-exporter/Chart.yaml:13]: I know this is a bit pedantic and annoying, but I'd put your information in as a maintainer as well, which will allow us to know you are the local maintainer and anyone (like infra) knows they should come to you to ask questions. Alternatively, you could put the SRE team down as a maintainer.

**Review by notjames — CHANGES_REQUESTED** [review]
see comment

File: helm-chart-sources/mysql-exporter/Chart.yaml (hunk 1 above)
Comment (thangarajbluescape) [reply helm-chart-sources/mysql-exporter/Chart.yaml:13]: Sure @notjames , Will do that.

Comment (thangarajbluescape) [reply helm-chart-sources/mysql-exporter/Chart.yaml:15]: Done @notjames

**Comment by thangarajbluescape** [comment]
No longer required as upstream accepted our changes.

**Pull request closed without merging**